src/views/SettingsPage.vue

```diff
@@ -142,6 +142,13 @@
             </div>
           </form>
         </section>
+        <section class="fields" v-else>
+          <h1>Add New User</h1>
+          <h4>
+            Current account does not have access to adding new users. Please
+            contact system administrator.
+          </h4>
+        </section>
```

Comment:
Maybe just hide this entire section because those who don't have permission to add users don't need to care about this.